### DISCUSSION TITLE
6 digit number for price

### DISCUSSION THREAD

**Discussion by kocsgab** [rgriebl/brickstore#462] [Q&A]
Hi All,

I use the currency in HUF, but I can only add 5 digit numbers for the price.
Is there any way to change it to 6 digit number?

Regards,
Gabor

**Comment by rgriebl** [comment]
You can't change it. However, I just added a feature to the current dev build to automatically add (or remove) significant digits, depending on the document's currency exchange rate.

**Reply by kocsgab** [reply]
In Build 558 I can make a 7 digit price. Thank you.
